src/plugins/timelist/Timelist.vue

```diff
@@ -110,14 +110,17 @@ export default {
     },
     mounted() {
         this.isEditing = this.openmct.editor.isEditing();
-        this.timestamp = Date.now();
+        this.timestamp = this.openmct.time.clock()?.currentValue || this.openmct.time.bounds()?.start;
```

Comment:
Isn't `clock().currentValue` a function?

Reply:
Excellent catch @shefalijoshi, ticket is updated!